openhands_resolver/github_issue.py

```diff
@@ -7,3 +8,6 @@ class GithubIssue(BaseModel):
     number: int
     title: str
     body: str
+    closing_issues: Optional[list[str]] = []
+    review_comments: Optional[list[str]] = []
+    head_branch: Optional[str] = None
```

Comment:
```suggestion
    closing_issues: list[str] | None = None
    review_comments: list[str] | None = None
    head_branch: str | None = None
```
I think `None` is OK here as the default to indicate that it hasn't been initialized (to distinguish with the case where it has been initialized).